pkgs/development/python-modules/gower/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  setuptools,
+  fetchPypi,
+  scipy,
+  pandas,
+}:
+
+buildPythonPackage rec {
+
+  pname = "gower";
+  version = "0.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-NN21FY8Oi/ugk9yga5+Ie9okSZjRCvKjrYx0pu+htfY=";
+  };
+
+  patchPhase = ''
+    rm pyproject.toml
+  '';
+
+  pythonImportsCheck = [ "gower" ];
+
+  dependencies = [
+    setuptools
+    scipy
+    pandas
+  ];
+
```

Comment:
Could you enable tests with `pytestCheckHook`?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook-using-pytestcheckhook

Reply:
it does have unit tests: https://github.com/wwwjk366/gower/blob/master/tests/matrix_test.py
I don't know why pytestCheckHook does not find those

Reply:
@natsukium should we skip the `pytestCheckhook`?

Reply:
You were right, the PyPi tarball didn’t include the tests, but the code on github did!